### DISCUSSION TITLE
Why is version 5.6.x tagged _next_ on npm?

### DISCUSSION THREAD

**Discussion by fcollonval** [lerna/lerna#3354] [Q&A] [answered]
Dear maintainers,

Is the branch 5.6.x not yet stable? Why is it released with the tag `next` and not `latest` on npm?

**Comment by JamesHenry** [comment, marked as answer]
We were just performing some additional testing before exposing to everyone by default. It did become the version on `latest`, although at the time of writing the `latest` version is now v6
